GameData/RealismOverhaul/Engine_Configs/F1_Config.cfg

```diff
@@ -126,3 +129,30 @@
 		maxAmount = 1
 	}
 }
+
+@PART[*]:HAS[@MODULE[ModuleEngineConfigs]:HAS[@CONFIG[F-1]],!MODULE[TestFlightInterop]]:BEFORE[zTestFlight]
+{
+	TESTFLIGHT
+	{
+		name = F-1
+		ratedBurnTime = 165					// Source 6
+		ignitionReliabilityStart = 0.93		// There were never really any true failures of the F-1 though a couple times an engine burned out early
+		ignitionReliabilityEnd = 0.988
+		cycleReliabilityStart = 0.94
+		cycleReliabilityEnd = 0.985
+	}
+}
+
+@PART[*]:HAS[@MODULE[ModuleEngineConfigs]:HAS[@CONFIG[F-1A]],!MODULE[TestFlightInterop]]:BEFORE[zTestFlight]
+{
+	TESTFLIGHT
+	{
+		name = F-1A
+		ratedBurnTime = 180					// No data, but if throttable, let's say it would last longer
```

Comment:
Maybe instead of arbitrarily making the burn time longer because it *can* throttle, instead use the feature of TestFlight that lets TF scale the burn time *based on* current throttle?